imports/plugins/core/system-info/server/no-meteor/schemas/schema.graphql

```diff
@@ -0,0 +1,9 @@
+extend type Query {
+  systemInformation: SystemInfo
+}
+
+"Represents Reaction System Infomation"
+type SystemInfo {
+  # core api version
+  apiVersion: String
+}
```

Comment:
I think it would be nice to add some comments here about other fields we plan to perhaps add to this type in the future as per Eric's sketch, just so someone else implementing them will know for sure they are going in the right spot.